viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1772,14 +1783,22 @@ export class Cognite3DViewer {
 
   private getCustomObjectIntersectionIfCloser(
     pixelCoords: THREE.Vector2,
-    closestDistanceToCamera: number | undefined
+    useDepthTest: boolean,
+    closestDistanceToCamera: number | undefined,
+    predicate?: (event: ICustomObject) => boolean
   ): CustomObjectIntersection | undefined {
```

Comment:
I think this would be cleaner if we made the second argument an `options`-object that contained all of these options